Corona-Warn-App/src/main/java/de/rki/coronawarnapp/worker/BackgroundNoiseOneTimeWorker.kt

```diff
@@ -38,6 +40,7 @@ class BackgroundNoiseOneTimeWorker @AssistedInject constructor(
             }
         }
 
+        Timber.d("$id: doWork() finished with %s", result)
```

Comment:
I there a reason you do not write Timber.d("$id: doWork() finished with **$result**")?

Reply:
Doesn't matter here I think, but the difference would be:

The solution with `%s` using string formatting would defer the `toString` evaluation until it's actually needed. So in the scenario where `result` is a complex object (i.e. a list of complex objects), we would not evaluate it `toString()` if logging is disabled (internal optimization in Timber).